domain/schema/model/sql/0024-command-block.sql

```diff
@@ -0,0 +1,24 @@
+CREATE TABLE block_command_type (
+    id INT PRIMARY KEY,
+    name_type TEXT NOT NULL
+);
+
+CREATE UNIQUE INDEX idx_block_command_type_name
+ON block_command_type (name_type);
+
+INSERT INTO block_command_type VALUES
+(0, 'destroy'),
+(1, 'remove'),
+(2, 'change');
+
+CREATE TABLE block_command (
+    uuid TEXT PRIMARY KEY,
```

Comment:
Might even be inclined to omit this, but we'll leave it for now.

Reply:
We need it for the API... No idea why because you can't query using it! This will at least keep it consistent for now and I don't want to bump the facade.